docs/reuse/common-blocks.md

```diff
@@ -0,0 +1,44 @@
+Start: Install Pebble binary
+
+1. Visit the [latest release page](https://github.com/canonical/pebble/releases/latest) to get the latest tag.
+2. Run the following commands to download the file. Make sure to replace `v1.12.0` with the latest tag and `amd64` with your architecture.
```

Comment:
There are three commands here that download, extract, and install, but then you have them repeated again in steps 3 and 4. I'd suggested remove them here, and just say "Run the following command ..." (singular).